### PR TITLE
More changes to breaking changes doc

### PR DESCRIPTION
**Pull request by ajcvickers**
Common format; release milestones included; fewer headings.

### CODE REVIEW

File: entity-framework/core/what-is-new/ef-core-3.0/breaking-changes.md [hunk 1]
```diff
@@ -374,31 +532,112 @@ protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
...
+
+**Why**
+
+This change was made because EF Core uses `SQLitePCLRaw.bundle_e_sqlite3` by default, which in turn means that FK enforcement is switched on by default and does not need to be explicitly enabled each time a connection is opened.
+
+**Mitigations**
+
+Foreign keys are enabled by default in SQLitePCLRaw.bundle_e_sqlite3, which is used by default for EF Core. For other cases, foreign keys can be enabled by specifying `Foreign Keys=True` in your connection string.
+
+## Microsoft.EntityFrameworkCore.Sqlite now depends on SQLitePCLRaw.bundle_e_sqlite3
```
Comment (divega) [inline entity-framework/core/what-is-new/ef-core-3.0/breaking-changes.md]: Are we conflating Microsoft.EntityFrameworkCore.Sqlite changes with Microsoft.Data.Sqlite? If so, is it on purpose?

File: entity-framework/core/what-is-new/ef-core-3.0/breaking-changes.md [hunk 2]
```diff
@@ -374,31 +532,112 @@ protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
...
+
+**Why**
+
+This change was made so that the version of SQLite used on iOS consistent with other platforms.
+
+**Mitigations**
+
+To use the native SQLite version on iOS, configure `Microsoft.Data.Sqlite` to use a different `SQLitePCLRaw` bundle.
+
+## Breaks that should only impact database providers
```
Comment (divega) [inline entity-framework/core/what-is-new/ef-core-3.0/breaking-changes.md]: Maybe add a note at to top of the document instead of having this here?

Comment (ajcvickers) [reply entity-framework/core/what-is-new/ef-core-3.0/breaking-changes.md]: @bricelam commented on the previous PR that these were intended to be listed as EF breaks.

File: entity-framework/core/what-is-new/ef-core-3.0/breaking-changes.md (hunk 1 above)
Comment (ajcvickers) [reply entity-framework/core/what-is-new/ef-core-3.0/breaking-changes.md]: @bricelam commented on the previous PR that these were intended to be listed as EF breaks.

File: entity-framework/core/what-is-new/ef-core-3.0/breaking-changes.md (hunk 2 above)
Comment (ajcvickers) [reply entity-framework/core/what-is-new/ef-core-3.0/breaking-changes.md]: I'm fine for you to move it where you want, or even delete it.

File: entity-framework/core/what-is-new/ef-core-3.0/breaking-changes.md (hunk 1 above)
Comment (divega) [reply entity-framework/core/what-is-new/ef-core-3.0/breaking-changes.md]: Yeah, I saw it just now. Thanks.

**Pull request merged**